akka-remote/akka-profiling-hifreq.jfc

```diff
@@ -0,0 +1,824 @@
+        </event>
+
+      </category>
+
+      <category label="Tcp">
+
+        <category label="Inbound">
+
+          <event name="akka.remote.artery.jfr.TcpInboundBound" label="Bound">
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+          </event>
+
+          <event name="akka.remote.artery.jfr.TcpInboundConnected" label="Connected">
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          </event>
+
+          <event name="akka.remote.artery.jfr.TcpInboundReceived" label="Received">
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+          </event>
+
+        </category>
+
+        <category label="Outbound">
+
+          <event name="akka.remote.artery.jfr.TcpOutboundSent" label="Sent">
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+          </event>
+
+          <event name="akka.remote.artery.jfr.TcpOutboundConnected" label="Connected">
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          </event>
+
+        </category>
+
+      </category>
+
+    </category>
+
```

Comment:
I tried with Aeron. It worked great for MaxThroughputSpec and LatencySpec.

Please add the following also:
```
      <category label="Aeron">

        <category label="Sink">
          <event name="akka.remote.artery.jfr.AeronSinkDelegateToTaskRunner" label="DelegateToTaskRunner">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSinkReturnFromTaskRunner" label="ReturnFromTaskRunner">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSinkEnvelopeOffered" label="Offered">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSinkEnvelopeGrabbed" label="Grabbed">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
        </category>

        <category label="Source">
          <event name="akka.remote.artery.jfr.AeronSourceDelegateToTaskRunner" label="DelegateToTaskRunner">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSourceReturnFromTaskRunner" label="ReturnFromTaskRunner">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSourceReceived" label="Received">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
          <event name="akka.remote.artery.jfr.AeronSourceReceived" label="Received">
            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
          </event>
        </category>

      </category>
```